src/components/FreakTile/__test__/FreakTile.test.jsx

```diff
@@ -4,15 +4,27 @@ import FreakTile from '../FreakTile';
 
 describe('FreakTile', () => {
   it('should render the freak tile', () => {
-    render(<FreakTile id={ 5 } picture="some-url" name="naruto" />);
+    render(<FreakTile id={ 5 } picture="some-url" name="name" />);
 
     expect(screen.getByTestId('freak-tile')).toBeInTheDocument();
   });
 
-  it('should render an hyperlink to the freak page', () => {
+  it('should render the tile as a hyperlink', () => {
     const id = 6;
```

Comment:
You used 5 as an ID everywhere except here, you chose 6? Stands out in a weird way. Be consistent.